xsite-replication/xsiteCache.xml

```diff
@@ -0,0 +1,11 @@
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsiteCache" statistics="true">
+      <backups>
+        <backup site="NYC" strategy="SYNC" failure-policy="WARN" timeout="12003">
+          <state-transfer chunk-size="098" timeout="7654" max-retries="321" wait-time="0101"/>
```

Comment:
this can be removed as well... too many random values xD

Reply:
the state transfert stuff ?

Reply:
you mean the backup property or the state transfert line

Reply:
the <state-transfer ..>

Reply:
ok